tooling/minitest/src/tests/int.rs

```diff
@@ -171,3 +171,166 @@ fn shr_works() {
     let p = p.finish_program(f);
     assert_stop(p);
 }
+
+#[test]
+fn unchecked_add_ub() {
+    let p = small_assume_program(eq(
```

Comment:
The `ValueExpr` needs to be executed somehow and `eq` is relatively self contained, such that we can fit it into `assume`. Even if it fails regardless.

Reply:
The `assume` is just as pointless though. This is confusing since on first reading it may seem like this tests that the `eq` fails and therefore the `assume` triggers UB, but actually it's a completely different UB you are testing for.

The helper is only used for UB tests, so IMO it would make more sense to have something like `assert_expr_ub` where you can just pass the `unchecked_add(const_int(i8::MAX), const_int(1_i8))`. The generated program should probably simply assign this expression to a local variable.

Reply:
Agree it is adding just unnecessary complexity. 
I added `assert_expr_ub` to the test library, so far a type for the local variable has still to be passed. Extracting it out of `ValueExpr` would be possible as an alternative.